ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -70,6 +59,22 @@ def sigmoid_output_transform(output):
     """
 
     def __init__(self, output_transform: Callable = lambda x: x, check_compute_fn: bool = False) -> None:
+        try:
+            from sklearn.metrics import cohen_kappa_score  # noqa: F401
+        except ImportError:
+            raise RuntimeError("This contrib module requires sklearn to be installed.")
+        self.precision_recall_curve_compute_fn = self.precision_recall_curve_compute()
```

Comment:
I don’t understand this line, it seems useless.

Reply:
Initially I also directly passed it to the super function but then I was taking reference for this from the file cohen_kappa.py where this was followed, so I thought may be there's some significance.

Reply:
I see. Although you didn't do exactly the same. To do so, you should define a wrapper in your function as it's done in cohen_kappa.py.

Reply:
I see. The function returns a wrapper, let's do it like that.